src/pluginUpdater.php

```diff
@@ -204,8 +204,14 @@ public function request_is_activate( $field_setting ) {
 			$json_data = json_decode( $body, true );
 			if ( isset( $json_data['success'] ) && $json_data['success'] === true ) {
 				return true;
+			} elseif ( isset( $json_data['errors'] ) ) {
+					$this->error_messages = nl2br( $this->generateErrorMessage( $json_data['errors'] ) );
+					return false;
+			} elseif ( isset( $json_data['message'] ) ) {
+				$this->error_messages = nl2br( $this->generateErrorMessage( $json_data['message'] ) );
+				return false;
 			} else {
-				$this->error_messages = nl2br( $this->generateErrorMessage( $json_data['errors'] ) ); // 'Something went wrong: ' . print_r( $json_data['errors'], true );
+				$this->error_messages = nl2br( $this->generateErrorMessage( $json_data ) );
```

Comment:
When will this line execute? Will $json_data have messages in this case? It seems to me it contains the full body of the response. Do we want to print that content as a message?

Reply:
are you right, it will print response and this we don't need it for users .